tests/schema/test_fields.py

```diff
@@ -147,3 +147,29 @@ def c(self):
         "b": 2,
         "c": 3.4,
     }
+
+
+def test_field_type_default():
+    @strawberry.type
+    class User:
+        name: str = "James"
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def a(self) -> User:
+            return User()
+
+    schema = strawberry.Schema(Query)
+
+    expected = """
+    type Query {
+      a: User!
+    }
+    
+    type User {
+      name: String!
+    }
+    """
+
+    assert print_schema(schema) == textwrap.dedent(expected).strip()
```

Comment:
added a test for normal strawberry types to make sure they have this behavior. didn't find a similar existing test like this. though lmk if there already is one.